rllib/algorithms/algorithm_config.py

```diff
@@ -561,6 +558,9 @@ def rollouts(
                 parallel sampling. Setting this to 0 will force rollouts to be done in
                 the local worker (driver process or the Algorithm's actor when using
                 Tune).
+            worker_index: Used internally for Policies to be aware of which rollout
```

Comment:
**question:** What is this? which worker_index? Isn't worker index set inside the WorkerSet? How come user has control over it now?

Reply:
why can't you just leave it where it was instead of hardcoding it to 0 in the user config and making it explicit? Ideally the policy constructor should get a separate worker_index / num_worker kwarg instead of getting it through the config object.

Reply:
ok, per our offline chat, reverted the change to the top-level config dict.
also updated PR description.